doc/source/whatsnew/v2.1.0.rst

```diff
@@ -67,6 +67,40 @@ Also, note that :meth:`Categorical.map` implicitly has had its ``na_action`` set
 This has been deprecated and will :meth:`Categorical.map` in the future change the default
 to ``na_action=None``, like for all the other array types.
 
+.. _whatsnew_210.enhancements.enhancement3:
+
+Index now has diff() and round() methods
```

Comment:
A one line mention under the `Other Enhancements` section is sufficient

Reply:
My mistake, this one should be fine